File: engine/model/src/main/scala/tools/packet/showitemgaininchat/ShowIntro.scala
package tools.packet.showitemgaininchat

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ShowIntro(private var _path: String) extends PacketInput {
  def path: String = _path

  override def opcode(): SendOpcode = SendOpcode.SHOW_ITEM_GAIN_INCHAT
}

File: engine/engine-model/src/main/java/tools/packet/showitemgaininchat/ShowIntro.java
package tools.packet.showitemgaininchat;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ShowIntro(String path) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.SHOW_ITEM_GAIN_IN_CHAT;
   }
}